ruff.toml

```diff
@@ -0,0 +1,19 @@
+target-version = "py310"
+
+ignore = ["ANN101", "ANN002", "ANN003", "ANN401"]
+
+  
+
+[lint]
+
+extend-select = [
```

Comment:
there are so many extensions 🤯 is `flake8-annotations` not in there by default the way `pydocstyle` apparently is?